x=2
try:
    print(x)

except:
    print("Error")